toolkit/tools/imagecustomizerapi/user.go

```diff
@@ -17,6 +17,7 @@ type User struct {
 	PasswordPath        string   `yaml:"PasswordPath"`
 	PasswordExpiresDays *int64   `yaml:"PasswordExpiresDays"`
 	SSHPubKeyPaths      []string `yaml:"SSHPubKeyPaths"`
+	SSHPubKeys          []string `yaml:"SSHPubKeys"`
```

Comment:
This is image customizer's public interface. The image modifier is free to reuse these types if it makes sense to do so. But this API can't be changed without also applying the same functionality to the image customizer.

Reply:
got it. then I will also add same functionality to the image customizer to allow this in the API